hadoop-ozone/interface-client/src/main/proto/OMAdminProtocol.proto

```diff
@@ -57,7 +57,7 @@ message OMNodeInfo {
 /**
  The service for getting OM metadata.
```

Comment:
Minor: Can we change doc to have something like service for OM admin operations??